src/Pyramid.js

```diff
@@ -0,0 +1,67 @@
+import {assign, constant} from "d3plus-common";
+
+import {default as Plot} from "./Plot";
+
+/**
+    @class Pyramid
+    @extends Plot
+    @desc Creates a population pyramid based on an array of data.
+    @example <caption>the equivalent of calling:</caption>
+new d3plus.Plot()
+  .discrete("y")
+  .shape("Bar")
+  .stacked(true)
+  .xConfig({tickFormat: d => Math.abs(d)})
+*/
+export default class Pyramid extends Plot {
+
+  /**
+      @memberof Pyramid
+      @desc Invoked when creating a new class instance, and overrides any default parameters inherited from Plot.
+      @private
+  */
+  constructor() {
+    super();
+
+    this._discrete = "y";
+    this._shape = constant("Bar");
+    this._stacked = true;
+    this._xConfig = {
+      tickFormat: d => Math.abs(d)
+    };
+
+  }
+
+  /**
+      Extends the draw behavior of the abstract Viz class.
+      @private
+  */
+  _draw(callback) {
+    this._filteredData.map((d, i) => ({
+      __d3plus__: true,
+      data: assign(d, {
+        __d3plusPopulation__:
+          this._groupBy[0](d, i) === "female"
+            ? this._x(d, i) * -1
+            : this._x(d, i)
+      }),
+      i,
+      hci: this._confidence && this._confidence[1] && this._confidence[1](d, i),
+      id: this._ids(d, i)
+        .slice(0, this._drawDepth + 1)
+        .join("_"),
+      lci: this._confidence && this._confidence[0] && this._confidence[0](d, i),
+      shape: this._shape(d, i),
+      x:
+        this._groupBy[0](d, i) === "female"
+          ? this._x(d, i) * -1
+          : this._x(d, i),
+      y: this._y(d, i)
+    }));
```

Comment:
this static referencing of "female" will not work, as we cannot rely on users to always be using "male" and "female" for the group by.

how about this solution: instead of creating a new `__d3plusPopulation__` variable, modify the non-discrete accessor to flip the number negative if it is part of the "second" group (the group whose first index in the data array is not `0`). Then, this is really just a stacked bar chart diverging from zero. We'll also need to override the tickFormat for the non-discrete axis to use the absolute value.

Here's a quick prototype: http://jsfiddle.net/xhczum43/4/